lib/tizen_sdk.dart

```diff
@@ -95,11 +95,12 @@ class TizenSdk {
           ? 'package-manager-cli.exe'
           : 'package-manager-cli.bin');
 
-  File get certificateProfilesFile =>
+  // This method will be used in [TpkTarget] for cache validation
```

Comment:
Please remove this. I was actually talking about the PR comment but not an in-line comment.